src/views/navigation.html

```diff
@@ -0,0 +1,24 @@
+{% from "macros/svg-elements.html" import singleColourIcon as singleColourIcon %}
+{% if not (requestedUrl.path === '/') %}
+  <a class="icon" columns="1.5" rows="1.0" rel="prev" href="/">
+    {{ singleColourIcon(asset, 'Home') }}<br>
+    Home
+  </a>
+{% endif %}
+{# dashboard view explicitly sets is_dashboard to true, otherwise is falsey #}
+{% if not (requestedUrl.path === '/apply/dashboard') %}
+  <a class="icon" columns="1.5" rows="1.0" rel="prev" href="/apply/dashboard">
+    {{ singleColourIcon(asset, 'Dashboard') }}<br>
+    Dashboard
+  </a>
+{% endif %}
+{% if (moment().isBefore(event.dates.getApplicationsEnd()) and not (requestedUrl.path === '/apply/form')) %}
```

Comment:
Do we want to show this to hackers who have already finished their application?

Reply:
This is fixed in #544